OpenRA.Mods.Common/Widgets/Logic/ServerCreationLogic.cs

```diff
@@ -180,7 +180,7 @@ void CreateAndJoin()
 			var name = Settings.SanitizedServerName(panel.Get<TextFieldWidget>("SERVER_NAME").Text);
 			int listenPort;
 			if (!Exts.TryParseIntegerInvariant(panel.Get<TextFieldWidget>("LISTEN_PORT").Text, out listenPort))
-				listenPort = 1234;
+				listenPort = Game.Settings.Server.ListenPort;
```

Comment:
We can then add a check and specific "Port must be between 49151 and 65535" error dialog here.

Reply:
we could put a warning and implement a method which is using 1234 as alternate port